modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -21,13 +21,40 @@ object SeqexecEngine {
 
   sealed trait SeqexecEvent
   object SeqexecEvent {
-    case class SequenceStart(view: SequenceView) extends SeqexecEvent
-    case class StepExecuted(view: SequenceView) extends SeqexecEvent
-    case class SequenceCompleted(view: SequenceView) extends SeqexecEvent
-    case class StepBreakpointChanged(view: SequenceView) extends SeqexecEvent
-    case class StepSkipMarkChanged(view: SequenceView) extends SeqexecEvent
-    case class SequencePauseRequested(view: SequenceView) extends SeqexecEvent
-    case class NewLogMessage(msg: LogMsg) extends SeqexecEvent
+
+    import edu.gemini.seqexec.engine
+    import edu.gemini.seqexec.engine.Event
+
+    case class SequenceStart(view: List[SequenceView]) extends SeqexecEvent
```

Comment:
This is *very* provisional. The idea here is to have something that can be used directly on the client as it is, so that we can show a quick demo. Isn't a `List[SequenceView]` what the client expects as state from the server?

The equivalent to `List[SequenceView]` has a distinct type named just `Queue`. But I was thinking that we could take care of this in the next iteration after watching the demo.